src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type CommentProps = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<CommentProps> = ({ comment }) => {
+  const { name, email, body } = comment;
```

Comment:
This could also be destructured in the component's parameters on line 8:
<img width="527" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/87986955/695d9bdf-b172-4025-a279-49b3396752d8">

Reply:
I moved destructuration to component's parameter.